client/app/scripts/superdesk-monitoring/monitoring.spec.js

```diff
@@ -114,23 +114,23 @@ describe('monitoring', function() {
         it('can get criteria for file type filter', inject(function(cards) {
             var card = {_id: '123', fileType: JSON.stringify(['text'])};
             var criteria = cards.criteria(card);
-            expect(criteria.source.post_filter.and).toContain({
-                terms: {type: ['text']}
+            expect(criteria.source.query.filtered.filter.and).toContain({
+                terms: {'type': ['text']}
             });
         }));
 
         it('can get criteria for file type filter with search', inject(function(cards) {
             var card = {_id: '123', fileType: JSON.stringify(['text']), query: 'test'};
             var criteria = cards.criteria(card);
-            expect(criteria.source.post_filter.and).toContain({
-                terms: {type: ['text']}
+            expect(criteria.source.query.filtered.filter.and).toContain({
+                terms: {'type': ['text']}
             });
         }));
 
         it('can get criteria for multiple file type filter', inject(function(cards) {
             var card = {_id: '123', fileType: JSON.stringify(['text, picture'])};
             var criteria = cards.criteria(card);
-            expect(criteria.source.post_filter.and).toContain({
+            expect(criteria.source.query.filtered.filter.and).toContain({
                 terms: {type: ['text, picture']}
```

Comment:
shouldn't it be this way type `['text', 'picture']` instead of `['text, picture']`